### PR TITLE
Document DS disabled functionality when connected to FMS

### LINKED ISSUE #814: DS spacebar & enter key functionality
The section "Driver Station Key Shortcuts" [1] doesn't warn the reader that when the DS is connected to FMS, enter and spacebar keypresses are ignored, and do not disable or e-stop the robot. [2][3]  When safety controls violate UI design best practices (e.g. by being mode-dependent and therefore non-habituating) [4] this is especially important to document.

Suggestion:
_Note: When connected to FMS during a match, teams must press the E-stop button in the Player Station to emergency stop their robot, as the enter and spacebar keys are ignored by the FMS._

[1] https://docs.wpilib.org/en/stable/docs/software/driverstation/driver-station.html#driver-station-key-shortcuts
[2] https://www.chiefdelphi.com/t/driver-station-ghost-estop/157662/28
[3]
image
[4] http://nitpicker.pbworks.com/w/page/12451253/The%2520Humane%2520Interface

### PR DESCRIPTION
**Pull request by Starlight220**
Resolves #814

### CODE REVIEW

**Comment by Starlight220** [comment]
@Daltz333 this is ready for review, though the wording could use some polishing - review it harshly.

File: source/docs/software/driverstation/driver-station.rst [hunk 1]
```diff
@@ -177,4 +177,6 @@ Driver Station Key Shortcuts
   * `Enter` - Disable the Robot
   * `Space` - Emergency Stop the robot. After an emergency stop is triggered the roboRIO will need to be rebooted before the robot can be enabled again.
 
-.. Note:: Space bar will E-Stop the robot regardless of if the Driver Station window has focus or not
+.. note:: Space bar will E-Stop the robot regardless of if the Driver Station window has focus or not
+
+.. warning:: When connected to FMS in a match, teams mush press the Team Station E-Stop button to emergency stop their robot as FMS ignores DS enable/disable and E-Stop signals and key shortcuts.
```
[the remark this answers is not in the document]
Comment (sciencewhiz) [reply source/docs/software/driverstation/driver-station.rst:182]: To be technically accurate, the FMS doesn't ignore, it's the DS that ignores the keyboard commands when connected to FMS.

[the remark this answers is not in the document]
Comment (Starlight220) [reply source/docs/software/driverstation/driver-station.rst:182]: How about this?
```suggestion
.. warning:: When connected to FMS in a match, teams mush press the Team Station E-Stop button to emergency stop their robot as the DS enable/disable and E-Stop key shortcuts are ignored.
```

[the remark this answers is not in the document]
Comment (Daltz333) [reply source/docs/software/driverstation/driver-station.rst:182]: I think that's acceptable

[the remark this answers is not in the document]
Comment (sciencewhiz) [reply source/docs/software/driverstation/driver-station.rst:182]: mush->must

[the remark this answers is not in the document]
Comment (Starlight220) [reply source/docs/software/driverstation/driver-station.rst:182]: So
```suggestion
.. warning:: When connected to FMS in a match, teams must press the Team Station E-Stop button to emergency stop their robot as the DS enable/disable and E-Stop key shortcuts are ignored.
```

[the remark this answers is not in the document]
Comment (Daltz333) [reply source/docs/software/driverstation/driver-station.rst:182]: Yes

**Review by Daltz333 — APPROVED** [review]

**Pull request merged**